bash_completion/kerl

```diff
@@ -7,89 +7,123 @@
 # shellcheck disable=SC2312  # Consider invoking this command separately to avoid masking its return value (or use '|| true' to ignore)
 
 _kerl() {
-    local cur prev
-    if type _get_comp_words_by_ref &>/dev/null; then
-        _get_comp_words_by_ref cur prev
-    else
-        cur=$2 prev=$3
-    fi
+    local command cur prev releases builds installations file
+
+    command=${COMP_WORDS[1]}
+    cur=${COMP_WORDS[${#COMP_WORDS[@]} - 1]}
+    prev=${COMP_WORDS[${#COMP_WORDS[@]} - 2]}
```

Comment:
This change seems to be adapted to our needs, without sacrificing readability or maintainability. It also allows some of the checks below to work as intended (e.g. the `else` in the build would not be possible since if `$prev` was `build` `COMP_CWORD` had to be 2.